bin/ghe-backup

```diff
@@ -41,8 +41,13 @@ done
 # shellcheck source=share/github-backup-utils/ghe-backup-config
 . "$( dirname "${BASH_SOURCE[0]}" )/../share/github-backup-utils/ghe-backup-config"
 
+# Check to make sure moreutils parallel is installed
+ghe_parallel_check
+
 # Used to record failed backup steps
 failures=
+tempdir="$(mktemp -d -t backup-utils-backup-XXXXXX)"
```

Comment:
I don't think you need a whole directory as everything is going to a single file. Why not go straight for the temp file like this?

```suggestion
failures_file="$(mktemp backup-utils-backup-failures-XXXXXX)"
```

You can then remove the next line and update the var in `cleanup()`.

Reply:
Done